examples/plotting/ImageAnimatorWCS_Example.py

```diff
@@ -0,0 +1,40 @@
+"""
+================
+ImageAnimatorWCS
+================
+
+This example shows a simple visualization for an ImageAnimatorWCS object
+"""
+import matplotlib.pyplot as plt
+from itertools import product
+import numpy as np
+
+import astropy
+import astropy.units as u
+
+import sunpy
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE, AIA_193_IMAGE
+from sunpy.visualization.animator import ImageAnimatorWCS
+
+###############################################################################
+# Creating the Mapcube of the given images
+sequence = sunpy.map.Map(AIA_171_IMAGE, AIA_193_IMAGE, sequence=True)
+map_sequence = sequence.as_array()
+
+###############################################################################
+# We roll the axis backwards to make the shape (10, 4096, 4096)
+# The last two axes are the image
+map_sequence = np.rollaxis(map_sequence, -1)
+
+wcs_input_dict = {f'{key}{n+1}': map_sequence.all_meta()[0].get(f'{key}{n}')
+                  for n, key in product([1, 2], ['CTYPE', 'CUNIT', 'CDELT'])}
+t0, t1 = map(parse_time, [k['date-obs'] for k in map_sequence.all_meta()])
+time_diff = (t1 - t0).to(u.s)
+wcs_input_dict.update({'CTYPE1': 'Time', 'CUNIT1': time_diff.unit.name, 'CDELT1': time_diff.value})
+
+WCS = astropy.wcs.WCS(wcs_input_dict)
+
+###############################################################################
+# Generate animation object with corresponding tick labels and tick postions
+ImageAnimatorWCS(map_sequence, wcs=WCS, vmax=1000).update_axes_style((20 * u.arcsec), 't', 'r')
```

Comment:
I am not sure I would make this one line. 

If we split it, maybe have two figures showcasing the function.